orchestra/todos/urls.py

```diff
@@ -12,9 +11,9 @@
 
 urlpatterns = [
     url(r'^todo/$',
-        TodoList.as_view(), name='todos'),
+        TodoViewset.as_view({'post': 'create', 'get': 'list'}), name='todos'),
     url(r'^todo/(?P<pk>[0-9]+)/$',
```

Comment:
to make sure i get it: does the dict you're passing in restrict the endpoints to those of the original viewset?